stable/prometheus-operator/templates/prometheus/servicemonitor.yaml

```diff
@@ -21,6 +21,13 @@ spec:
     {{- if .Values.prometheus.serviceMonitor.interval }}
     interval: {{ .Values.prometheus.serviceMonitor.interval }}
     {{- end }}
+    {{- if .Values.prometheus.serviceMonitor.scheme }}
+    scheme: {{ .Values.prometheus.serviceMonitor.scheme }}
+    {{- end }}
+    {{- if .Values.prometheus.serviceMonitor.tlsConfig }}
+    tlsConfig:
+    {{ toYaml .Values.prometheus.serviceMonitor.tlsConfig | nindent 6 }}
```

Comment:
I believe you want to use indent, instead of nindent, because you're already starting on a newline.

Reply:
I moved it to the line above otherwise you have to put indent at the start of the line which is ugly. Good spot.